### PR TITLE
order expiry time added fixes issue #5159

### LINKED ISSUE #5159: Allow event organiser to specify expiry time for an order
**Is your feature request related to a problem? Please describe.**

I propose to allow the event organizer to set the expiry time for an order form.
default value will be 10 minutes

**Additional context**

Parent: #4852

### PR DESCRIPTION
**Pull request by starlord1311**
Fixes #5159 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Changes proposed in this pull request:

-
-
-

### CODE REVIEW

File: app/api/schema/events.py [hunk 1]
```diff
@@ -254,6 +254,7 @@ def validate_timezone(self, data, original_data):
                                 schema='CustomFormSchema',
                                 many=True,
                                 type_='custom-form')
+    order_expiry_time = fields.Float(default=10)
```
Comment (srv-twry) [inline app/api/schema/events.py]: Keep the attributes above the relationship declaration. Also add `allow_none=True`.

File: app/models/event.py [hunk 2]
```diff
@@ -148,6 +148,7 @@ class Event(SoftDeletionModel):
                             primaryjoin='UsersEventsRoles.event_id == Event.id',
                             secondaryjoin='User.id == UsersEventsRoles.user_id',
                             backref='events')
+    order_expiry_time = db.Column(db.Float, default=10)
```
Comment (srv-twry) [inline app/models/event.py]: Keey the attributes above the relationships.

**Comment by starlord1311** [comment]
@srv-twry what to update in the docs?

**Comment by srv-twry** [comment]
@starlord1311 See, you have added a new field in the events model. So our developers should know that a new field has been added in the events model. It is very important specially for the frontend and android devs.

Check the api blueprint file and search for `Group Events` and add this field to samples there.

**Comment by starlord1311** [comment]
@srv-twry can you tell how to migrate databases

**Comment by starlord1311** [comment]
screenshot from 2018-07-23 22-49-00
@srv-twry any idea what to do now?

**Comment by bhaveshAn** [comment]
Hey, since you aren't using virtual environment so please install dependencies using `sudo pip3 install -r requirements.txt` and let us know on this.

**Comment by starlord1311** [comment]
@bhaveshAn @srv-twry can you explain how to test changes using postman?

**Comment by srv-twry** [comment]
@starlord1311 
I am sorry but I have to close this PR and add this by myself. Actually this is required right now for one of the features and can't wait.

**Pull request closed without merging**